### DISCUSSION TITLE
关于GSEA的疑问

### DISCUSSION THREAD

**Discussion by zhaoliang0302** [ShixiangWang/self-study#55] [Q&A] [answered]
翔哥你好，我分析TCGA后找了个感兴趣的lncRNA，现在想通过GSEA看看它参与了什么信号通路，我根据这个lncRNA表达量分成了高低两组（两组sample数量相同），然后对这两组DEseq得到了差异基因再用clusterprofiler完成GSEA，疑惑的是：
1.根据这个lncRNA分成高低两组的cutoff是否是median？
2.DEseq后是要选择logFC>1（或2...）还是所有的都保留，pvalue需要<0.05吗？
谢谢

**Comment by ShixiangWang** [comment, marked as answer]
cutoff设为均值或上下25%比较常见，具体分组策略应该从自己对研究的理解出发。

做GSEA可以将所有基因保留，然后按照p值或logFC排序，用排序的基因列表做GSEA。具体参考y叔写的clusterProfiler包

**Comment by zhaoliang0302** [comment]
好的，logFC没有要求，分组获得的差异基因的p值是否要小于0.05呢？
我是看到这篇文章的 https://www.jianshu.com/p/dcd5725d6908

**Comment by ShixiangWang** [comment]
@zhaoliang0302 看你自己了，做不做都可以。关键是表达排序。

**Comment by ShixiangWang** [comment]
@zhaoliang0302 没问题我先关闭了，有问题你再打开。
